.docker/source/Dockerfile

```diff
@@ -29,6 +29,6 @@ RUN \
 # Build the workspace
 RUN . /opt/ros/${ROS_DISTRO}/setup.sh &&\
     colcon build \
-            --cmake-args -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
-              --ament-cmake-args -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+            --cmake-args -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
```

Comment:
In the past, I found it very handy to have MoveIt binaries with debug symbols around.
Why do you want to change from RelWithDebInfo? This is essentially with all release optimizations, but with debug symbols, so larger but now slower.

Reply:
The build for `foxy-source` is failing with the following after `93 min`

```
Unexpected error
Encountered error: ('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))
Traceback (most recent call last):
File "/stage/builder/runner.py", line 297, in _run
self.build()
File "/stage/builder/runner.py", line 221, in build
self._build()
File "/stage/builder/runner.py", line 209, in _build
cache_repo)
File "/stage/builder/build.py", line 43, in build_image
for line in stream:
File "/usr/local/lib/python2.7/dist-packages/docker/api/client.py", line 290, in _stream_helper
for chunk in json_stream(self._stream_helper(response, False)):
File "/usr/local/lib/python2.7/dist-packages/docker/utils/json_stream.py", line 66, in split_buffer
for data in stream_as_text(stream):
File "/usr/local/lib/python2.7/dist-packages/docker/utils/json_stream.py", line 22, in stream_as_text
for data in stream:
File "/usr/local/lib/python2.7/dist-packages/docker/api/client.py", line 296, in _stream_helper
data = reader.read(1)
File "/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/response.py", line 311, in read
flush_decoder = True
File "/usr/lib/python2.7/contextlib.py", line 35, in __exit__
self.gen.throw(type, value, traceback)
File "/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/response.py", line 244, in _error_catcher
raise ProtocolError('Connection broken: %r' % e, e)
ProtocolError: ('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))
```

Looking in the [autonated build limits](https://hub.docker.com/support/doc/docker-hub-automated-build-fails-and-the-logs-are-missing-empty) it means we don't have timeout here, so either we're running out of RAM (which I don't think is the case since I have an automated docker image build for MoveIt2 + MTC for a private project and it working pretty well) or we're running out of disk space (which building with `RelWithDebInfo` may cause)

Reply:
This flag `-DBUILD_TESTING=OFF` will disable building tests

Reply:
Sorry, I missed this option. Obviously, I was too tired. LGTM then.